sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1686,4 +1689,162 @@ class HiveDDLSuite
       }
     }
   }
+
+  Seq("a b", "a:b", "a%b").foreach { specialChars =>
```

Comment:
please add a TODO to remove these duplicated tests when we merge DDLSuite and HiveDDLSuite